controllers/provider-gcp/example/Infrastructure.yaml

```diff
@@ -0,0 +1,42 @@
+---
+apiVersion: extensions.gardener.cloud/v1alpha1
+kind: Cluster
+metadata:
+  name: default
+spec:
+  cloudProfile:
+    apiVersion: garden.sapcloud.io/v1beta1
+    kind: CloudProfile
+  seed:
+    apiVersion: garden.sapcloud.io/v1beta1
+    kind: Seed
+  shoot:
+    apiVersion: garden.sapcloud.io/v1beta1
+    kind: Shoot
+    spec:
+      cloud:
+        gcp:
+          networks:
+            pods: 10.243.128.0/17
+            services: 10.243.0.0/17
+    status:
+      lastOperation:
+        state: Succeeded
+
+---
+apiVersion: extensions.gardener.cloud/v1alpha1
+kind: Infrastructure
+metadata:
+  namespace: default
```

Comment:
This example probably won't work as it would try to create a VPC with name `default` that already exists in GCP.
Also, could you add here an how an example `Secret` should look like (and for AWS as well, it's missing there) so that users know the needed keys?